src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx

```diff
@@ -71,6 +72,12 @@ const BrowserTrackConfig: FunctionComponent<BrowserTrackConfigProps> = (
     } else {
```

Comment:
line 69 (`Object.keys(browserCogTrackList).map...`) — instead of map we should have used a forEach.

Reply:
That's done